src/CookiePlugin.php

```diff
@@ -109,11 +109,12 @@ private function createCookie(RequestInterface $request, $setCookie)
 
             switch (strtolower($key)) {
                 case 'expires':
-                    $expires = \DateTime::createFromFormat(DATE_COOKIE, $value);
+                    $expires = \DateTime::createFromFormat(DATE_COOKIE, $value)->getTimestamp()
```

Comment:
> That would mean that the expires header sets, the actual expires header in the Cookie object (8th param).

If the expires header is actually set, I think it should be set in the Cookie object as well. Otherwise it is inconsistent. Whether it is deprecated or not, we should follow the cookie specifications. Also, the expires header itself is deprecated, not the expires property of the object.

> It also would be a change in behaviour compared to the current master version.

Why would it? The expires header is a datetime object per specification/documentation. Changing it to integer value seems to be a change in behavior.

Or am I missing something?